lib/process-services-cloud/src/lib/form/components/widgets/properties-viewer/properties-viewer.widget.ts

```diff
@@ -47,6 +47,16 @@ export class PropertiesViewerWidgetComponent extends WidgetComponent {
         super(formService);
     }
 
+    ngOnInit(): void {
+        if (this.field &&
```

Comment:
Why not adding the input property to pass the file id / node id we want to open up? this seems that we are forcing a little bit our way through . We already merged a similar PR but please apply the same changes also to the other one

Reply:
After performing some investigations and having discussions between us, we decided that the currently implemented approach will fulfill our needs without any problem. 

_P.S._ by design the value coming to this component is an array of files not a single file. any changes to this will require a big code refactoring.